### PR TITLE
Adds more topics to manifest file

### PR DESCRIPTION
**Pull request by pamelipluas**
**Topics:**

```
---
# Source: mw-data-processor/templates/topics.yaml
---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "bay.processor.catalog.v1"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2

---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "bay.processor.products.v1"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2

---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "bay.processor.products.by.skn.v1"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2

---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "mw-data-processor-app-price-state-store-changelog"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2

---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "mw-data-processor-app-promo-state-store-changelog"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2

---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "mw-data-processor-app-product-state-tore-changelog"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2


```

### CODE REVIEW

File: charts/mw-data-processor/values.yaml [hunk 1]
```diff
@@ -42,6 +42,14 @@ kafka:
         name: "bay.processor.catalog.${version}"
         partitions: 1
         replicas: 2
+      catalogProcessor:
+        name: "bay.processor.catalog.sink.${version}"
```
Comment (josehbc) [inline charts/mw-data-processor/values.yaml]: delete

File: charts/mw-data-processor/values.yaml [hunk 2]
```diff
@@ -42,6 +42,14 @@ kafka:
...
+        partitions: 1
+        replicas: 3
+      productProcessorSink:
```
Comment (josehbc) [inline charts/mw-data-processor/values.yaml]: delete

File: charts/mw-data-processor/values.yaml [hunk 3]
```diff
@@ -54,8 +62,17 @@ kafka:
         name: "bay.processor.products.${version}"
         partitions: 1
         replicas: 2
+      priceStateStore:
```
Comment (josehbc) [inline charts/mw-data-processor/values.yaml]: the correct names for internal topics are:
mw-data-processor-app-promo_state_store-changelog
mw-data-processor-app-price_state_store-changelog
mw-data-processor-app-product_state_tore-changelog

Comment (pamelipluas) [reply charts/mw-data-processor/values.yaml]: updated

**Comment by pamelipluas** [comment]
With the latest changes:

```
---
# Source: mw-data-processor/templates/topics.yaml
---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "bay.processor.catalog.v1"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2

---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "bay.processor.products.v1"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2

---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "bay.processor.products.by.skn.v1"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2

---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "mw-data-processor-app-price-state-store-changelog"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2

---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "mw-data-processor-app-promo-state-store-changelog"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2

---
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaTopic
metadata:
  name: "mw-data-processor-app-product-state-tore-changelog"
  labels:
    strimzi.io/cluster: kafka-cluster
spec:
  partitions: 1
  replicas: 2


```

could you please take a look again @josehbc thanks :)

**Pull request merged**